lib/iruby/command.rb

```diff
@@ -22,7 +37,26 @@ def run
 
     private
 
+    def ipython_register_file
+      @ipython_register_file ||= @ipython_dir / "kernels" / "ruby" / "kernel.json"
+    end
+
+    def print_help
+      puts <<-EOF.chomp
+Usage:
+    iruby register        Register IRuby kernel into #{@ipython_dir}.
+    iruby unregister      Remove #{ipython_register_file}.
+    iruby console         Launch the IRuby terminal-based console.
+    iruby notebook        Launch the IRuby HTML notebook server.
+    ...                   Same as IPython.
+
+Please note that IRuby accepts the same parameters as IPython.
+Try `ipython help` for more information.
+EOF
+    end
+
     def run_kernel
+      raise "Currently `iruby kernel` can only be called by a frontend." if @args.size == 1
       raise(ArgumentError, 'Not enough arguments to the kernel') if @args.size < 2 || @args.size > 4
```

Comment:
Since IPython supports standalone kernel, i.e., running kernel without a frontend. I think it's better to notice users that IRuby doesn't support it **currently**.